docs/ui/automations.md

```diff
@@ -70,14 +88,83 @@ Similarly, if a trigger fires on work queue health and the action is to pause an
 
-## Coming soon
+## Automation notifications
+
+Notifications enable you to set up automation actions that send a message. 
+
+Automation notifications support sending notifications via any predefined block that is capable of and configured to send a message. That includes, for example:
+
+- Slack message to a channel
+- Microsoft Teams message to a channel
+- Email to a configured email address
+
+![Configuring notifications for an automation in Prefect Cloud.](/img/ui/automations-notifications.png)
+
+!!! note "Notification blocks must be pre-configured"
+    Notification blocks must be pre-configured prior to creating a notification action. Any existing blocks capable of sending messages will be shown in the block drop-down list.
+
+    The **Add +** button cancels the current automation creation process and enables configuration a notification block.
+
+## Templating notifications with Jinja
+
+The notification body can include templated variables using [Jinja](https://palletsprojects.com/p/jinja/) syntax. Templated variable enable you to include details relevant to automation trigger, such as a flow or queue name. 
+
+Jinja templated variable syntax wraps the variable name in double curly brackets, like `{{ variable }}`.
+
+You can access properties of the underlying flow run objects including:
+
+- [flow_run](/api-ref/orion/schemas/core/#prefect.orion.schemas.core.FlowRun)
+- [flow](/api-ref/orion/schemas/core/#prefect.orion.schemas.core.Flow)
+- [deployment](/api-ref/orion/schemas/core/#prefect.orion.schemas.core.Deployment)
+- [work_queue](/api-ref/orion/schemas/core/#prefect.orion.schemas.core.WorkQueue)
+
+In addition, each object includes an `id` UUID and `created` and `updated` timestamps. 
+
+The `flow_run|ui_url` token returns the URL for viewing the flow run in Prefect Cloud.
+
+Here’s an example for something that would be relevant to a flow run state-based notification:
+
+```
+Flow run {{ flow_run.name }} entered state {{ flow_run.state.name }}. 
+
+    Timestamp: {{ flow_run.state.timestamp }}
+    Flow ID: {{ flow_run.flow_id }}
+    Flow Run ID: {{ flow_run.id }}
+    State message: {{ flow_run.state.message }}
+```
+
+The resulting Slack webhook notification would look something like this:
+
+![Configuring notifications for an automation in Prefect Cloud.](/img/ui/templated-notification.png)
+
+You could include flow and deployment details.
+
+```
+Flow run {{ flow_run.name }} for flow {{ flow.name }}
+entered state {{ flow_run.state.name }}
+with message {{ flow_run.state.message }}
+
+Flow tags: {{ flow.tags }}
+Deployment name: {{ deployment.name }}
+Deployment version: {{ deployment.version }}
+Deployment parameters: {{ deployment.parameters }}
+```
+
+An automation reporting on work queue health might include notifications using work queue details.
+
+```
+Work queue health alert!
+
+Name: {{ work_queue.name }}
+Last polled: {{ work_queue.last_polled }}
+```
```

Comment:
These are solid examples, and I think they'll be a familiar soft landing for folks familiar with Prefect but not with the Events system (pretty much everyone).  What do you think about including something like this (paraphrasing):

In addition to those shortcuts for flows, deployments, and work queues, you have access to the automation and the event that triggered your automation:

```
Automation: {{ automation.name }}
Description: {{ automation.description }}

Event: {{ event.id }}
Resource:
  {% for label, value in event.resource %}
  {{ label }}: {{ value }}
  {% %}
Related Resources:
  {% for related in event.related %}
    Role: {{ related.role }}
    {% for label, value in event.resource %}
    {{ label }}: {{ value }}
    {% %}
  {% endfor %}
```

Then something something Events API: https://app.stg.prefect.dev/api/docs#tag/Events/operation/create_workspace_events_api_accounts__account_id__workspaces__workspace_id__events_post

Reply:
That also teases some more of the Jinja2 capabilities, which could help folks who are familiar with it

Reply:
Below the fold is probably the right place for it :D  It's kind of in the weeds